src/porepy/fracs/fractures_2d.py

```diff
@@ -479,6 +480,27 @@ def plot(self, **kwargs):
         """
         pp.plot_fractures(self.domain, self.pts, self.edges, **kwargs)
 
+    def to_csv(self, file_name):
```

Comment:
I agree, this is something we should have.

Reply:
:D great! thanks